gtk4-macros/src/lib.rs

```diff
@@ -12,3 +14,12 @@ pub fn composite_template_derive(input: TokenStream) -> TokenStream {
     let gen = composite_template_derive::impl_composite_template(&input);
     gen.into()
 }
+
+#[proc_macro_attribute]
+#[proc_macro_error]
+pub fn template(attrs: TokenStream, item: TokenStream) -> TokenStream {
```

Comment:
Can you add some documentation for the macro?

Reply:
I'll definitely do that